lib/transport/selenium-webdriver/method-mappings.js

```diff
@@ -709,6 +719,56 @@ module.exports = class MethodMappings {
 
           return null;
         },
+
+        async contextClick(webElementOrId) {
```

Comment:
@beatfactor what do you think about this?

Reply:
It would be too confusing. The left click action is not an alias of the click command. The click command is clicking an element, whereas the left click action is commonly used to perform a click on a position in the screen.

Reply:
cool. I think for now this change would do be enough then